https://issues.redhat.com/browse/ACM-6269 (#5276)
open-cluster-management/rhacm-docs: clusters/release_notes/whats_new.adoc

- For convenience, you can add your pull secret and SSH public key to your host inventory credential, then automatically populate those fields by selecting the cluster when creating a cluster in an on-premises environment. See xref:../clusters/credential_on_prem.adoc#creating-a-credential-for-an-on-premises-environment[Creating a credential for an on-premises environment] for more information.  

> - ClusterCurator now supports specifying an Ansible inventory to use for all jobs, and this is available to {mce-short} users in the console as well. See xref:../clusters/cluster_lifecycle/ansible_config_cluster.adoc#specify-ansible-inventory[Specifying an Ansible inventory to use for all jobs] for more information. 

Reviewer: ```suggestion
- ClusterCurator now supports specifying an Ansible inventory to use for all jobs, and this is also available to {mce-short} users in the console. See xref:../clusters/cluster_lifecycle/ansible_config_cluster.adoc#specify-ansible-inventory[Specifying an Ansible inventory to use for all jobs] for more information. 
```

> - ClusterCurator now supports specifying an Ansible inventory to use for all jobs, and this is available to {mce-short} users in the console as well. See xref:../clusters/cluster_lifecycle/ansible_config_cluster.adoc#specify-ansible-inventory[Specifying an Ansible inventory to use for all jobs] for more information. 

Reviewer 2: I checked the issue and didn't see a developer issue related to it. We do update What's new if it is feature work. I would ask for the dev issue for this feature before adding to What's new. 
```suggestion
- Use the `ClusterCurator` resource to specify an Ansible inventory to use for all jobs. This is also available to {mce-short} users in the console as well. See xref:../clusters/cluster_lifecycle/ansible_config_cluster.adoc#specify-ansible-inventory[Specifying an Ansible inventory to use for all jobs] for more information. 
```

Author: Here are the related dev issues: https://issues.redhat.com/browse/ACM-4436

https://issues.redhat.com/browse/ACM-3842

I'll add them to the current issue and update this PR tomorrow. Thanks for the review!
